core/templates/dev/head/pages/about-page/about-page.scripts.ts

```diff
@@ -16,6 +16,6 @@
  * @fileoverview Directive scripts for the about page.
  */
 
-require('base_components/BaseContentDirective.ts');
+require('App.ts');
```

Comment:
This script is already included in about-page.controller.ts. So this wasn't needed.

Also, now I think that this script require is more suited here than about-page.controller.ts since it is directly used in the mainpage

Reply:
Added the script in here and removed from about-page.controller.ts file